src/modules/practice-client-intakes/http.ts

```
@@ -1,16 +1,46 @@
 import { OpenAPIHono } from '@hono/zod-openapi';
+import { getLogger } from '@logtape/logtape';
 import { zValidator } from '@hono/zod-validator';
```

ESLint

[error] `@hono/zod-validator` import should occur before import of `@logtape/logtape`

(import/order)